### DISCUSSION TITLE
util missing from pushkin directory?

### DISCUSSION THREAD

**Discussion by IrisxBi** [l3atbc-datadog/pushkin#93] [General] [answered]
Hi! Basic question- I'm just following the quickstart guide, and don't seem to be getting the util folder within the pushkin directory, or the seeds folder within each of the basic experiments. Will this cause any issues down the road?

**Comment by parkerrobbins** [comment, marked as answer]
Thanks for catching that! Our illustrations of the directory structures are outdated. We will update those right away!

**Comment by restmegan** [comment]
Hi @IrisxBi ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
